File: src/com/xwkj/form/domain/Reserve.java
package com.xwkj.form.domain;

import java.io.Serializable;
import java.util.Date;

public class Reserve implements Serializable {
	private static final long serialVersionUID = 2692727044095071731L;
	
	private String rid;
	private String name;
	private String telephone;
	private String address;
	private String service;
	private Integer area;
	private String user;
	private Integer number;
	private Date createDate;
	
	public String getRid() {
		return rid;
	}
	public void setRid(String rid) {
		this.rid = rid;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getTelephone() {
		return telephone;
	}
	public void setTelephone(String telephone) {
		this.telephone = telephone;
	}
	public String getAddress() {
		return address;
	}
	public void setAddress(String address) {
		this.address = address;
	}
	public String getService() {
		return service;
	}
	public void setService(String service) {
		this.service = service;
	}
	public Integer getArea() {
		return area;
	}
	public void setArea(Integer area) {
		this.area = area;
	}
	public String getUser() {
		return user;
	}
	public void setUser(String user) {
		this.user = user;
	}
	public Integer getNumber() {
		return number;
	}
	public void setNumber(Integer number) {
		this.number = number;
	}
	public Date getCreateDate() {
		return createDate;
	}
	public void setCreateDate(Date createDate) {
		this.createDate = createDate;
	}

}

File: WebContent/static/js/reserve.js
$(document).ready(function() {
	$("#reserve-submit").click(function() {
		var name=$("#reserve-name").val();
		var telephone=$("#reserve-telephone").val();
		var address=$("#reserve-address").val();
		var service=$("input[name='reserve-service']:checked").val();
		var area=$("#reserve-area").val();
		var user=$("input[name='reserve-user']:checked").val();
		var number=$("#reserve-number").val();
		if(name==""||telephone==""||address==""||service==""||user=="") {
			alert("请填写所有带星号的必填项目！");
			return;
		}
		ReserveManager.add(name, telephone, address, service, area, user, number, function(rid) {
			if(rid) {
				alert("提交成功");
				location.href="index.html";
			}
		});
	});
});